src/Socket.php

```diff
@@ -680,15 +683,20 @@ public static function showListForItem($item)
        // Link to open a new socket
         if ($item->getID() && self::canCreate()) {
             echo "<div class='firstbloc'>";
-            Html::showSimpleForm(
-                Socket::getFormURL(),
-                '_add_fromitem',
-                __('New socket for this item...'),
-                [
-                    '_from_itemtype' => $item->getType(),
-                    '_from_items_id' => $item->getID(),
-                ]
-            );
+            echo "\n<form method='get' action='" . Socket::getFormURL() . "'>\n";
+            echo "<input type='hidden' name='items_id' value='" . $item->getID() . "'>\n";
+            echo "<input type='hidden' name='itemtype' value='" . $item->getType() . "'>\n";
+            echo "<div class='firstbloc'><table class='tab_cadre_fixe'>\n";
+            echo "<tr class='tab_bg_2'>";
+            echo "<td class='tab_bg_2 center'>";
+            echo "<td>\n";
+            echo "<input type='submit' name='add_several' value=\"" . __('New socket for this item...') . "\" class='btn btn-primary'>\n";
```

Comment:
1) You should use a `<button>` here, to prevent value to be added in URL parameters.
2) You should use `_from_items_id` name, to not break previous behaviour.

Reply:
fixed